src/components/tl/ucp/reduce_scatter/reduce_scatter_knomial.c

```diff
@@ -214,6 +216,7 @@ void ucc_tl_ucp_reduce_scatter_knomial_progress(ucc_coll_task_t *coll_task)
     size_t                    data_size       = count * dt_size;
     ucc_rank_t                rank            = task->subset.myrank;
     ucc_rank_t                size            = task->subset.map.ep_num;
+    ucc_rank_t                root            = 0;
```

Comment:
nit: does it have a real meaning to set root at 0 here and below ?

Reply:
root is set to 0 to handle the case when reduce scatter is part of allreduce SRA schedule, in this case we don't need ranks remapping which is equivalent to using root 0 in VRANK mapping